components/service/glean/src/test/java/mozilla/components/service/glean/GleanTest.kt

```diff
@@ -4,12 +4,10 @@
 
 package mozilla.components.service.glean
 
-import android.content.Context
 import androidx.lifecycle.Lifecycle
-import androidx.lifecycle.LifecycleOwner
 import androidx.lifecycle.LifecycleRegistry
-import androidx.test.core.app.ApplicationProvider
-import androidx.work.testing.WorkManagerTestInitHelper
+import androidx.test.ext.junit.runners.AndroidJUnit4
```

Comment:
nit let's not change this, if possible. This will produce an additional change when porting things over to the new repo :)